app/components/forms/orders/order-form.js

```diff
@@ -37,8 +37,8 @@ export default Component.extend(FormMixin, {
   }),
   sameAsBuyer: false,
 
-  getRemainingTime: computed('data', function() {
-    let orderExpiryTime = this.get('data.event.orderExpiryTime');
+  getRemainingTime: computed('settings', function() {
```

Comment:
The computed property should be observing everything you are using. `settings.orderExpiryTime`, `data.createdAt` and `data.identifier`.

Please pay more attention to detail